src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -1262,23 +1263,38 @@ LottieLayer* LottieParser::parseLayers()
     root->type = LottieLayer::Precomp;
     root->comp = comp;
 
+    Array<LottieLayer*> pendingLayers;
```

Comment:
Now, this is becoming more than a parsing job. Please double-check the `_buildComposition()` function in `tvgLottieBuilder.cpp` I think that place is more appropriate to handle this.